docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_suite_from_a_json_schema_file.rst

```diff
@@ -0,0 +1,91 @@
+.. _how_to_guides__how_to_create_a_suite_from_a_json_schema_file:
+
+How to create a new Expectation Suite from a jsonschema file
+============================================================
+
+
+The ``JsonSchemaProfiler`` helps you quickly create :ref:`Expectation Suites` from jsonschema files.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+  - :ref:`Set up a working deployment of Great Expectations <getting_started>`
+  - Have a valid jsonschema file
+
+Steps
+-----
+
+1. **Set a filename and a suite name**
+
+    .. code-block:: python
+
+        jsonschema_file = "YOUR_JSON_SCHEMA_FILE.json"
+        suite_name = "YOUR_SUITE_NAME"
+
+2. **Load a DataContext**
+
+    .. code-block:: python
+
+        context = ge.data_context.DataContext()
+
+3. **Load the jsonschema file**
+
+    .. code-block:: python
+
+        with open(jsonschema_file, "r") as f:
+            raw_json = f.read()
+            schema = json.loads(raw_json)
+
+4. **Instantiate the profiler**
+
+    .. code-block:: python
+
+        profiler = JsonSchemaProfiler()
+
+5. **Create the suite**
+
+    .. code-block:: python
+
+        suite = profiler.profile(schema, suite_name)
+
+6. **Save the suite**
+
+    .. code-block:: python
+
+        context.save_expectation_suite(suite)
+
+7. **Generate Data Docs and review the results there**
+
+    .. code-block:: bash
+
+        great_expectations docs build
+
+You can iterate on the Expectations to get dial in the Suite you want using ``great_expectations suite edit``.
+
+Additional notes
+----------------
+
+.. important::
+
+    Note that there is not yet a notion of nested data types in Great Expectations so suites generated use column map expectations.
+
+The full example script is here:
+
+.. code-block:: python
+
+    import json
+    import great_expectations as ge
+    from great_expectations.profile.json_schema_profiler import JsonSchemaProfiler
+
+    jsonschema_file = "YOUR_JSON_SCHEMA_FILE.json"
+    suite_name = "YOUR_SUITE_NAME"
+
+    context = ge.data_context.DataContext()
+
+    with open(jsonschema_file, "r") as f:
+        raw_json = f.read()
+        schema = json.loads(raw_json)
+
+    print("Generating suite...")
+    profiler = JsonSchemaProfiler()
+    suite = profiler.profile(schema, suite_name)
+    context.save_expectation_suite(suite)
```

Comment:
Note as a follow on PR I'll need @kyleaton to show me how to wire up discuss.

Reply:
Nevermind! I stumbled upon the howto discuss howto guide! 🎉

Reply:
🎉